### PR TITLE
cli content added

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: modules/config/pages/eap-cli.adoc [hunk 1]
```diff
@@ -1,200 +1,329 @@
 = JBoss EAP CLI
 
-Connect to an instance of JBoss EAP standalone with the management CLI and execute general commands.
+* JBoss EAP 8 employs XML files for configuration, offering three methods:
 
-Management Command-line Interface (CLI) Tool Overview
-In Red Hat JBoss Enterprise Application Platform (JBoss EAP) the configuration values reside in XML files. There are three different options for configuring for JBoss EAP servers:
+. **Management console**: A web app for modifying XML configs.
+. **Management CLI**: A command-line tool for remote configuration and script writing.
+. **Manual XML editing**: Discouraged by Red Hat.
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc]: Reword - Generally discouraged to directly edit the XML config files, but may be needed in a few cases

File: modules/config/pages/eap-cli.adoc [hunk 2]
```diff
@@ -1,200 +1,329 @@
 = JBoss EAP CLI
 
-Connect to an instance of JBoss EAP standalone with the management CLI and execute general commands.
+* JBoss EAP 8 employs XML files for configuration, offering three methods:
 
-Management Command-line Interface (CLI) Tool Overview
-In Red Hat JBoss Enterprise Application Platform (JBoss EAP) the configuration values reside in XML files. There are three different options for configuring for JBoss EAP servers:
+. **Management console**: A web app for modifying XML configs.
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc]: Reword - JBoss EAP offers an easy to use web console to configure various subsystems of JBoss EAP

File: modules/config/pages/eap-cli.adoc [hunk 3]
```diff
@@ -1,200 +1,329 @@
...
+. **Management CLI**: A command-line tool for remote configuration and script writing.
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc]: Remove the word 'remote' - you can also connect locally

Comment (rsriniva) [reply modules/config/pages/eap-cli.adoc]: /s/script writing/scripting

File: modules/config/pages/eap-cli.adoc [hunk 4]
```diff
@@ -1,200 +1,329 @@
...
+Username: XXXXXXX
+Password: XXXXXXX
+[standalone@localhost:9990 /]
+----
+
+* To automatically connect, use the `--connect` or `-c` argument:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ $JBOSS_HOME/bin/jboss-cli.sh --connect
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc]: Can you check this again? connecting to localhost should not ask for creds?

File: modules/config/pages/eap-cli.adoc [hunk 5]
```diff
@@ -1,200 +1,329 @@
...
+[vagrant@server bin]$ $JBOSS_HOME/bin/jboss-cli.sh -c
+Authenticating against security realm: ManagementRealm
+Username: XXXXX
+Password: XXXXX
+[standalone@localhost:9990 /]
+----
+
+[Note]
+====
+Note: Authentication relies on the OS user running the management CLI being the same as the one starting JBoss EAP.
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc]: confusing and possibly incorrect. you authenticate using the user and pw you create with add-user.sh

File: modules/config/pages/eap-cli.adoc [hunk 6]
```diff
@@ -1,200 +1,329 @@
...
+----
+
+* Optionally you can use --user and --password to pass credentials if needed.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ $JBOSS_HOME/bin/jboss-cli.sh --connect  --user=XXXXX --password=XXXXX
+----
+
+=== JBoss EAP 8 Commands
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc:61]: can you convert this into an exercise?

**Review by rsriniva — APPROVED** [review]
Please address comments and merge.

File: modules/config/pages/eap-cli.adoc (hunk 4 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-cli.adoc]: correct, it does not need username and password when you run as jboss user

File: modules/config/pages/eap-cli.adoc (hunk 3 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-cli.adoc]: corrected

File: modules/config/pages/eap-cli.adoc (hunk 2 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-cli.adoc]: corrected

File: modules/config/pages/eap-cli.adoc (hunk 1 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-cli.adoc]: corrected

File: modules/config/pages/eap-cli.adoc (hunk 5 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-cli.adoc]: removed the note

**Pull request merged**